packages/cli/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@pipcook/cli",
-  "version": "2.1.1",
+  "version": "2.1.2-serve-beta.0",
```

Comment:
This version is informal, please use 2.1.2 :)

Reply:
It should be 2.1.1 unless we want to bump it.